specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,191 @@
+      - description: |
+                      Gegevens over het verblijf of de woonlocatie van een persoon.
+                      * **datumAanvangAdreshuishouding** : de datum van aangifte of ambtshalve melding van verblijf en adres.
+                      * **datumIngangGeldigheid** : datum waarop de gegevens over de verblijfplaats geldig zijn geworden.
+                      * **datumInschrijvingInGemeente**: bij inschrijving op grond van een verhuisaangifte door de burger is dit de aangiftedatum. Bij inschrijving op grond van een geboorteakte is dit de geboortedatum. Bij ambtshalve inschrijving is dit de datum waarop het voornemen van ambtshalve opneming schriftelijk aan de persoon is medegedeeld.
+                      * **datumVestigingInNederland** : datum van inschrijving in Nederland.
+                      * **landVanWaarIngeschreven** : het land waar de persoon woonde voor (her)vestiging in Nederland.
+                      * **gemeenteVanInschrijving** : de gemeente waar de persoon verblijft en is ingeschreven. De code kan voorloopnullen bevatten."
+        properties:
+          adresseerbaarObjectIdentificatie:
+            type: "string"
+            description: |
+                          De verblijfplaats van de persoon kan een ligplaats, een standplaats of een verblijfsobject zijn.
+            example: "0226010000038820"
+          aanduidingBijHuisnummer:
+            $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+          nummeraanduidingIdentificatie:
+            type: "string"
+            description: |
+                          Unieke identificatie van een nummeraanduiding (en het bijbehorende adres) in de BAG.
+            example: "0518200000366054"
+          functieAdres:
+            $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+          indicatieVestigingVanuitBuitenland:
+            type: boolean
+            description: |
+                          Geeft aan dat de ingeschreven persoon zich vanuit het buitenland heeft ingeschreven.
+          locatiebeschrijving:
+            type: "string"
+            description: |
+                          Omschrijving van de ligging van een verblijfsobject, standplaats of ligplaats.
+            example: "Naast de derde brug"
+          korteNaam:
+            description: |
+                          De officiële openbareruimtenaam uit de Basisregistratie Gebouwen en Adressen (BAG) of een verkorte versie.
+            type: string
+          vanuitVertrokkenOnbekendWaarheen:
+            type: boolean
+            description: |
+                          Geeft aan dat de persoon is teruggekeerd uit een situatie van 'vertrokken onbekend waarheen.'
+            example: true
+          datumAanvangAdreshouding:
```

Comment:
in v1.3 wordt deze gevuld uit óf datumAanvangAdreshouding óf Datum aanvang adres buitenland (afh. van welke van de twee gevuld is). Dit wordt door RvIG beschouwd als informatievraag.

Ik stel voor dat we los leveren datumAanvangAdreshouding en datumAanvangAdresBuitenland (deze dus toevoegen). En in de proxy hierbij toevoegen "datumVan" (in lijn met filter en veld datumTot bij historie) of "datumAanvangAdres", die gevuld wordt met óf datumAanvangAdreshouding en óf datumAanvangAdresBuitenland

Reply:
Is gedaan. Ik heb ook een voorstel gedaan om Verblijfplaats te splitsen naar een Verblijfplaats (Nederland) en VerblijfplaatsBuitenland. De meeste properties gelden volgens mij niet voor een buitenlands verblijfplaats en dat kunnen we nu expliciet definieren